### PR TITLE
Add NGINX non-iso timestamp format information

### PR DESCRIPTION
**Pull request by gr0**

### CODE REVIEW

File: docs/logs/supported-date-formats.md [hunk 1]
```diff
@@ -33,8 +33,12 @@ One of the key things when working with time based data is the timestamp that re
...
+6. NGINX non-ISO date format, for example:
+        
+        22/Apr/2016:14:40:13 +0200
+
 **Please note:**
 
   - if `@timestamp` field is not provided Sematext will set `@timestamp` to the time when the event is received by Sematext.
-  - if time specified in `@timestamp` does not include timezone
-    information, Logs App will assume UTC.
+  - if the `@timestamp` field is not parsable by Sematext, the `@timestamp` field will be set to teh time when the event is received by Sematext.
```
Comment (otisg) [inline docs/logs/supported-date-formats.md:43]: teh => the

File: docs/logs/faq.md [hunk 2]
```diff
@@ -614,6 +614,16 @@ example, if you send logs that are "2 hours ahead of real time" they
 will be accepted, but will not be displayed until 2 hours
 later.
 
+### What happens if I try to send data with @timestamp in a not supported format?
```
Comment (otisg) [inline docs/logs/faq.md:617]: What happens if I send data with non-supported @timestamp format?

**Review by otisg — CHANGES_REQUESTED** [review]

File: docs/logs/faq.md [hunk 3]
```diff
@@ -614,6 +614,15 @@ example, if you send logs that are "2 hours ahead of real time" they
...
+
+If your log events are sent to Sematext Cloud with a format that is not supported our
```
Comment (adnanrahic) [inline docs/logs/faq.md:619]:
```suggestion
If your log events are sent to Sematext Cloud with a format that is not supported in our 
```

File: docs/logs/supported-date-formats.md [hunk 4]
```diff
@@ -33,8 +33,12 @@ One of the key things when working with time based data is the timestamp that re
...
+  - if time specified in `@timestamp` does not include timezone information, Logs App will assume UTC.
```
Comment (adnanrahic) [inline docs/logs/supported-date-formats.md:44]:
```suggestion
  - if time specified in `@timestamp` does not include timezone information, the Logs App will assume UTC.
```

**Review by adnanrahic — APPROVED** [review]
Tiny edits and it's ready!

File: docs/logs/supported-date-formats.md (hunk 1 above)
Comment (gr0) [reply docs/logs/supported-date-formats.md:43]: Adjusted, thanks for catching that.

**Review by otisg — APPROVED** [review]

**Comment by gr0** [comment]
Thanks guys!

**Pull request merged**